client/notices.go

```diff
@@ -181,16 +206,24 @@ func makeNoticesQuery(opts *NoticesOptions) url.Values {
 	if opts == nil {
 		return query
 	}
+	if len(opts.UserIDs) > 0 {
+		for _, uid := range opts.UserIDs {
+			query.Add("user-ids", strconv.FormatUint(uint64(uid), 10))
+		}
+	}
 	if len(opts.Types) > 0 {
-		typeStrs := make([]string, 0, len(opts.Types))
 		for _, t := range opts.Types {
-			typeStrs = append(typeStrs, string(t))
+			query.Add("types", string(t))
```

Comment:
Ah, nice simplification, thanks! We don't actually need the `if len(opts.Types) > 0` any more with this change, as the loop won't execute at all if `opts.Types` is nil or empty. Same above and below.